.dockerignore

```diff
@@ -1,4 +1,3 @@
 inst/
 dist/
 build/
-.git/
```

Comment:
I wanted to copy the .git folder of forch so as to make it easier to install forch in the container. pbr needs git tags.

Reply:
Not a good idea to do it this way.  The .git directory can become arbitrarily large so can make things get crazy slow -- and esp doing this as a global wildcard.  If you need the git tags inside of the repo, you should do a clean checkout inside of the build container, e.g. https://github.com/grafnu/daq/blob/master/docker/modules/Dockerfile.test_bacnet#L10

Reply:
Makes sense. Though I wanted to move current codebase instead of git checkout so that we could test local changes that way. An alternative is to simply skip .git and do a git init and then let it carry the 0.0.0 version internally. If the version really matters, I could pass it as an argument some how. Do you thin the version inside the container is important?

Reply:
Resolved.